setup.cfg

```diff
@@ -1,2 +1,36 @@
+[metadata]
+name = pathspec
+version = 0.5.9
```

Comment:
The version is outdated. It should be "0.8.0.dev1".

Reply:
O shi, this shouldn't have been here. The version is meant to be set via git tags.